apps/aechannel/src/aesc_fsm.erl

```diff
@@ -936,55 +940,49 @@ awaiting_signature(cast, {?SIGNED, withdraw_tx, Tx} = Msg,
 awaiting_signature(cast, {?SIGNED, ?FND_CREATED, SignedTx} = Msg,
```

Comment:
It feels that the macros like `FND_CREATED` in aesc_codec.hrl (`-define(FND_CREATED  , funding_created).`) shall be shared with `sc_ws_api_jsonrpc` - as part of encoding/decoding in user API.

Though orthogonal to this PR.